java/src/jmri/jmrit/symbolicprog/LongAddrVariableValue.java

```diff
@@ -327,46 +314,36 @@ public void propertyChange(@Nonnull java.beans.PropertyChangeEvent e) {
             // busy transitions drive the state
             switch (_progState) {
                 case IDLE:  // no, just a CV update
-                    if (log.isDebugEnabled()) {
-                        log.error("Busy goes false with state IDLE");
-                    }
+                    log.error("Busy goes false with state IDLE");
                     return;
                 case READING_FIRST:   // read first CV, now read second
-                    if (log.isDebugEnabled()) {
-                        log.debug("Busy goes false with state READING_FIRST");
-                    }
+                    log.debug("Busy goes false with state READING_FIRST");
                     _progState = READING_SECOND;
                     highCV.read(_status);
                     return;
                 case READING_SECOND:  // finally done, set not busy
-                    if (log.isDebugEnabled()) {
-                        log.debug("Busy goes false with state READING_SECOND");
-                    }
+                    log.debug("Busy goes false with state READING_SECOND");
                     _progState = IDLE;
                     (_cvMap.get(getCvNum())).setState(READ);
                     highCV.setState(READ);
                     //super.setState(READ);
                     setBusy(false);
                     return;
                 case WRITING_FIRST:  // no, just a CV update
-                    if (log.isDebugEnabled()) {
-                        log.debug("Busy goes false with state WRITING_FIRST");
-                    }
+                    log.debug("Busy goes false with state WRITING_FIRST");
                     _progState = WRITING_SECOND;
                     highCV.write(_status);
                     return;
                 case WRITING_SECOND:  // now done with complete request
-                    if (log.isDebugEnabled()) {
-                        log.debug("Busy goes false with state WRITING_SECOND");
-                    }
+                    log.debug("Busy goes false with state WRITING_SECOND");
                     _progState = IDLE;
                     super.setState(STORED);
                     setBusy(false);
                     return;
                 default:  // unexpected!
                     log.error("Unexpected state found: {}", _progState);
                     _progState = IDLE;
-                    return;
+                    //return;
```

Comment:
Not sure why this return was commented out.  It's the parallel construction to all the other switch cases.

Reply:
As it is the last case, IntelliJ flags it as " 'return' unnecessary as the last statement in a 'void' method" (inside the if).

Reply:
Is there a way to turn that flagging off?

Lots of things are formally 'unnecessary' to the compiler:  No comments are necessary for a program to run, as just one example.  But something that's unnecessary can never-the-less add clarity for the reader.  

In this case, it causes extra cognitive load to have to figure out why that `return` isn't there, and others are.

Its also _not_ a parallel & independent construction:  If I add something after the case block, one case clause will suddenly start working differently, but only one.  The parallel & independent construction would be to have all the other case statements end with `break`, in which case the missing `break` (instead of `return`) would be a common idiom.